source/deploy/encryption-options.rst

```diff
@@ -31,6 +31,10 @@ The encryption uses AES-256 by default, and it is not configurable. However, it
 Encryption-at-rest
 ------------------
 
+Encryption-at-rest is available for messages via hardware and software disk encryption solutions applied to the Mattermost database, which resides on its own server within your infrastructure. To enable end user search and compliance reporting of message histories, Mattermost does not offer encryption within the database.
```

Comment:
> Mattermost does not offer encryption within the database.

We usually call this app-level encryption. 0/5 on keeping it as is or updating.

Reply:
@esarafianou - I'd appreciate your guidance here. Within our deployment guidance, are we trying to reach the widest audience possible, or are we aiming to resonate with a technical security-minded audience?

If widest audience possible, ``encryption-at-rest`` will likely resonate better to both technical and non-technical audiences, and it's more accessible and self-descriptive. 

``app-level encryption`` may be less familiar to those outside of technical or cybersecurity fields, as it is more specific and less commonly discussed in general contexts.

Reply:
That's a good point, Carrie. Let's leave it as is.